settings.gradle

```diff
@@ -26,8 +26,9 @@ include ':settings'
 include ':servicetest'
 include ':maps'
 include ':osmdroid'
+include ':icons'
 
 apply from: 'secrets.gradle'
 if (getSecrets().getProperty('MAPBOX_DOWNLOADS_TOKEN', '') != '') {
     include ':mapbox'
-}
+}
```

Comment:
Looks like there's a missing new line at the end of the file here.

Reply:
Why do we need that line?

Reply:
Git (and Github) warns against it (there's a good SO on why [here](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)). You can set up Android Studio to make sure you always have a new line in its settings (Editor > General > On Save > Ensure every saved file ends with a line break).

Reply:
Ok done.